packages/happner-2/test/integration/security/security-multiple-authproviders.js

```diff
@@ -1,115 +1,129 @@
-    let client = new test.Mesh.MeshClient();
-    try {
+    it('rejects login promise on bad credentials', async () => {
+      let client = new test.Mesh.MeshClient();
+      try {
+        await client.login({
+          ...testUser,
+          password: 'bad password',
+        });
+      } catch (e) {
+        test.expect(e.toString()).to.be('AccessDenied: Invalid credentials');
+      }
+    });
+
+    it('logs in correctly using happn3 auth provider', async () => {
+      let client = new test.Mesh.MeshClient();
       await client.login({
         ...testUser,
-        password: 'bad password',
+        authType: 'happn',
       });
-    } catch (e) {
-      test.expect(e.toString()).to.be('AccessDenied: Invalid credentials');
-    }
-  });
-
-  it('logs in correctly using happn3 auth provider', async () => {
-    let client = new test.Mesh.MeshClient();
-    await client.login({
-      ...testUser,
-      authType: 'happn',
     });
-  });
 
-  it('logs in correctly using second auth provider', async () => {
-    let client = new test.Mesh.MeshClient();
-    // eslint-disable-next-line no-useless-catch
-    try {
+    it('logs in correctly using second auth provider', async () => {
+      let client = new test.Mesh.MeshClient();
+      // eslint-disable-next-line no-useless-catch
       await client.login({
         ...testUser2, //uses default, i.e. 'second' authProvider
       });
-    } catch (e) {
-      throw e;
-    }
-  });
+    });
 
-  it('fails to log in when using second auth provider for happn3 type user', async () => {
-    let client = new test.Mesh.MeshClient();
-    try {
-      await client.login({
-        ...testUser, //uses default, i.e. 'second' authProvider
-      });
-      throw new Error('Should have thrown');
-    } catch (error) {
-      test.expect(error.toString()).to.eql('AccessDenied: Invalid credentials');
-    }
-  });
+    it('fails to log in when using second auth provider for happn3 type user', async () => {
+      let client = new test.Mesh.MeshClient();
+      try {
+        await client.login({
+          ...testUser, //uses default, i.e. 'second' authProvider
+        });
+        throw new Error('Should have thrown');
+      } catch (error) {
+        test.expect(error.toString()).to.eql('AccessDenied: Invalid credentials');
+      }
+    });
+
+    it('fails to log in when using happn3 auth provider for second type user', async () => {
+      let client = new test.Mesh.MeshClient();
+      try {
+        await client.login({
+          ...testUser2,
+          authType: 'happn',
+        });
+        throw new Error('Should have thrown');
+      } catch (error) {
+        test.expect(error.toString()).to.eql('AccessDenied: Invalid credentials');
+      }
+    });
 
-  it('fails to log in when using happn3 auth provider for second type user', async () => {
-    let client = new test.Mesh.MeshClient();
-    try {
+    it('Change password on a 2nd auth provider and check meta data', async () => {
```

Comment:
standard is lower case for test descriptions

Reply:
Fixed.